File: Python/Lista Enlazada/Lista_Enlazada.py
from Nodo import Persona

class Lista:
    def __init__(self):
        self.raiz = None
        
    def insertar(self, nombre, apellido, edad):
        
        Nodo_nuevo = Persona(nombre, apellido, edad)
        
        if self.raiz is None:
            self.raiz = Nodo_nuevo
        else:
            temporal = self.raiz
            while temporal.siguiente is not None:
                temporal = temporal.siguiente
            temporal.siguiente = Nodo_nuevo

    def imprimir(self):
        temporal = self.raiz
        while temporal != None:
            print('Nombre: '+ temporal.nombre +
                  '\nApellido: '+ temporal.apellido+
                  '\nEdad: '+ str(temporal.edad))
            temporal = temporal.siguiente

Prueba = Lista()
Prueba.insertar('Carlos', 'Rangel', 21)
Prueba.insertar('Luis', 'Castellanos', 20)
Prueba.imprimir()

File: Javacript/Lista Enlaza/Lista_Enlazada.js
class Nodo{
    constructor(id, nombre){
        this.id = id;
        this.nombre = nombre;
        this.siguiente = null;
    }
}

class ListaEnlazada{
    constructor(){
    this.raiz = null;
    }

    insertar(id, nombre){
        let nodo_nuevo = new Nodo(id, nombre)
        if (this.raiz == null){
            this.raiz = nodo_nuevo;
        }else{
            let temporal = this.raiz;
            while(temporal.siguiente != null){
                temporal = temporal.siguiente;
            }
            temporal.siguiente = nodo_nuevo;
        }
    }
 
    imprimir(){
        let temporal = this.first
        while(temporal != null){
            console.log('Id: ' + temporal.id +
                        '\nNombre: ' + temporal.nombre)
            temporal = temporal.siguiente
        }

    }



}
Prueba = new ListaEnlazada()
Prueba.insertar(1, 'Carlos')
Prueba.insertar(1, 'Katherine')
Prueba.imprimir()